config/routes.rb

```diff
@@ -13,6 +13,11 @@
         post 'rest'
       end
     end
+    resources :security_schemes do
+      collection do
```

Comment:
Oops I missed it on my first review, but you can get rid of lines 17 to 19 as you don't have a rest route on security schemes

Reply:
Thanks! Corrected